frontend/integration-tests-cypress/cypress/tests/monitoring.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { timeouts } from '../support/utils/timeouts';
```

Comment:
A thought on suites:

```typescript
// i know you only have one file, but many of our protractor test suites have multiple files.
// i believe cypress organizes suites just with a directory.
// what is tricky about this is some tests are included in multiple suites:
// - crudBasic - a single file from CRUD suite
// - crud - several files
// not sure if cypress encourages or discourages this in configuration in some way
integration-tests-cypress/suites/monitoring/monitoring.spec.ts
```

Reply:
`integration-tests-cypres/tests` is fine, as we have `integration-tests/tests`. (rather than rename to `suites`).

Reply:
nit: Do we need the extra `cypress` dir in `integration-tests-cypress/cypress`, or can we pull everything up one level?

Reply:
We used to need `integration-tests-cypress/cypress` when we had a separate `\node_modules` for the Cypress tests, since we moved everything to root, I was able to remove the internal `cypress` directory and point to new dir locations via the `cypress.json` config file.
New directory structure is:
![image](https://user-images.githubusercontent.com/12733153/80752878-532aea80-8afa-11ea-9ddf-b850afbf7c6f.png)
Created a `monitoring` test dir, displays nicely in test runner:
![image](https://user-images.githubusercontent.com/12733153/80752953-6f2e8c00-8afa-11ea-8764-975e21e69aaa.png)